BinaryTree.java

```diff
@@ -0,0 +1,165 @@
+package com.katermar;

```

Comment:
speaking about different branches for each assignment. can i just have single branch for current assignment? and to merge previous assignments to the master branch after reviews?